.studio/event-service

```diff
@@ -13,5 +13,5 @@ document "event_integration" <<DOC
   Run event service integration tests
 DOC
 function event_integration() {
-  go_test -v "github.com/chef/automate/components/event-service/integration_test"
+  go_test -v -tags=integration "github.com/chef/automate/components/event-service/integration_test"
```

Comment:
Based solely on the pattern I see of "integration tests should add this tag", I would think the same change should happen for these: 
.studio/secrets-service
.studio/pg-sidecar-service
.studio/ingest-service
.studio/event-feed-service
.studio/es-sidecar-service
.studio/config-mgmt-service
.studio/compliance-servic